src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -242,7 +242,7 @@ def register_submission(
 
             extra_data = get_unmapped_data(
                 submission, self.zaak_mapping, REGISTRATION_ATTRIBUTE
-            )
+            ).data
             # The extraElement tag of StUF-ZDS expects primitive types
             extra_data = flatten_data(extra_data)
```

Comment:
Note that this `flatten_data` would become unnecessary, as this flattening is already done by the `FormioData` structure